### ISSUE TITLE
Pixel 2 does not show camera

### ISSUE THREAD

**Issue by reubenkcoutinho** [google-ar/arcore-unity-sdk#45]
I've recently got a pixel 2 and I tried to load the AR Core demo app, but all I get is a blue screen. I followed the instructions to the letter. Any help would be appreciated

**Comment by chaosemer** [comment]
On Android O, there are issues with the developer preview not properly putting up the permissions dialog for camera permissions. Going to the App Info for your app and manually granting the camera permissions should address this.

**Comment by reubenkcoutinho** [comment]
Yes works now. This needs to be fixed , given that Google Pixel2 is all about having AR in it

**Comment by chaosemer** [comment]
I'm going to leave this open for other developers to know about.

Don't worry, we're on this for the 1.0 release.

**Comment by chaosemer** [comment]
This should now be resolved in Developer Preview 2. Please reopen if you still are having issues.

**Comment by jpark0906** [comment]
I have a similar issue: blue screen on my pixel 2 using the helloAR project through Unity. The apk would not build until I set the Camera Config filter in the inspector of the ARCore Device object. I've tried various options for the DefaultCameraConfigFilter settings but none seem to have an effect. The screen shows blue after starting and black dots will populate the screen. Occasionally I will see a surface detection grid.

![95145152_1491584374347978_8388917529703088128_n](https://user-images.githubusercontent.com/10178261/80873032-a91ea000-8c6a-11ea-9c9f-e0e31c6c8ae9.png)
